assets/.nvmrc

```diff
@@ -0,0 +1 @@
+12
```

Comment:
Do we need Node 12 or will things work with Node 14 as well?

Reply:
Updated to 14